superdesk/publish/http_push_tests.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+
+import json
+import os
+import unittest
+import requests
+
+from superdesk.publish import SUBSCRIBER_TYPES
+from superdesk.publish.http_push import HTTPPushService
+
+
+class ItemNotFound(Exception):
+    pass
+
+
+class HTTPPushPublishTestCase(unittest.TestCase):
+
+    def setUp(self):
+        super().setUp()
+
+        if 'HTTP_PUSH_RESOURCE_URL' not in os.environ:
+            return
+        self.resource_url = os.environ['HTTP_PUSH_RESOURCE_URL']
+
+        self.subscribers = [{"_id": "1", "name": "Test", "media_type": "media",
+                             "subscriber_type": SUBSCRIBER_TYPES.DIGITAL, "is_active": True,
+                             "sequence_num_settings": {"max": 10, "min": 1},
+                             "destinations": [{"name": "test", "delivery_type": "http_push", "format": "ninjs",
+                                               "config": {"resource_url": self.resource_url}
+                                               }]}]
+        self.formatted_item1 = {"_id": "item1",
+                                "headline": "headline",
+                                "versioncreated": "2015-03-09T16:32:23",
+                                "version": 1
+                                }
+        self.formatted_item2 = {"_id": "item1",
+                                "headline": "headline2",
+                                "versioncreated": "2015-03-21T13:43:51",
+                                "version": 2
+                                }
+        self.item = {'item_id': 'item1',
+                     'format': 'NITF',
```

Comment:
'ninjs' instead of 'NITF'?
Also it would be good to test image publishing as well.